### ISSUE TITLE
OCSP Stapling error

### ISSUE THREAD

**Issue by domharrington** [GUI/lua-resty-auto-ssl#56]
I've just gotten this error on our production server:

```
2017/04/07 19:05:21 [error] 595#595: *227 [lua] ssl_certificate.lua:212: set_cert(): auto-ssl: failed to set ocsp stapling for xxxxx.com - continuing anyway - failed to get ocsp response: failed to validate OCSP response (http://ocsp.int-x3.letsencrypt.org/): OCSP response not successful (6: unauthorized), context: ssl_certificate_by_lua*, client: x.x.x.x, server: 0.0.0.0:443\n
```

Have you seen this before? I'm on version **0.10.3-1**

**Comment by GUI** [comment]
Has the OCSP stapling error gone away, or are you still getting it? It looks like previously, this kind of "unauthorized" error was related to a Let's Encrypt outage (https://community.letsencrypt.org/t/ocsp-server-returns-unauthorized-status/21965), but I'm not sure if that's related this time around. OCSP stapling errors aren't fatal (your cert should still be returned and be valid), but if you're consistently getting these warnings, then it would be good to get the bottom of.

**Comment by domharrington** [comment]
I've not seen this error again since then, but at least once a day the instance hangs and i can see an error like this in the logs: https://github.com/GUI/lua-resty-auto-ssl/issues/48#issuecomment-288825700. Openresty then requires a restart.

**Comment by domharrington** [comment]
I've not gotten this error since. Our servers haven't had to be restarted since the fix in #48 too, so thanks for that!

**Comment by GUI** [comment]
Great! I'm glad to hear we got to the bottom of that and things have stabilized for you! Thanks again for the reports.
